app/views/search/_collection_list_item.html.erb

```diff
@@ -29,7 +29,7 @@
               <p>
                 <span class="panel-list-details" <% if is_project %>data-bind="style: {'background-color': $root.project.colorForCollection($data)}"<% end %>>
                   <span data-bind="text: short_name"></span>
-                  <!-- ko if: $data.version_id && $data.version_id !== 'Undefined' -->
+                  <!-- ko if: $data.version_id && $data.version_id !== 'Undefined' && $data.version_id !== 'Not provided' -->
```

Comment:
Oh, yeah, you are clear enough - I was just not listening; somehow didn't see the chat..
What should we ask CMR to provide to us if there is no version_id for a collection? I can write a ticket against CMR. 
So close it?